definitions/tools/bam_to_sam.cwl

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: 'BAM to SAM conversion'
+baseCommand: ["/opt/samtools/bin/samtools", "view", "-h", "-o"]
+requirements:
+    - class: DockerRequirement
+      dockerPull: "mgibio/samtools-cwl:1.0.0"
+    - class: ResourceRequirement
+      ramMin: 16000
+
+arguments: [out.sam]
```

Comment:
It might be nice to name this file based on the input file name like in `bam_to_cram.cwl`:
https://github.com/genome/analysis-workflows/blob/a23f42ef49c10a588fd35a3afaad5de03e253533/definitions/tools/bam_to_cram.cwl#L12